src/index.js

```diff
@@ -1,9 +1,17 @@
-// eslint-disable-next-line no-unused-vars
-import _ from 'lodash';
+import Score from './score.js';
 import './style.css';
 
-const component = () => {
+const newScore = new Score();
```

Comment:
- [ ] Kindly when the program starts call the newScore.fetchScore to view the data to the user in the beginning.

Reply:
Hi @ShahierNashaat ,
does this require to work when you click on refresh button?

Reply:
No you just call the function when the website loads and when click the refresh button